[2024-01-18T22:08:26.542+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.test_model_exp scheduled__2024-01-01T03:30:00+00:00 [queued]>
[2024-01-18T22:08:26.582+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.test_model_exp scheduled__2024-01-01T03:30:00+00:00 [queued]>
[2024-01-18T22:08:26.582+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:08:26.582+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-18T22:08:26.583+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:08:26.615+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): test_model_exp> on 2024-01-01 03:30:00+00:00
[2024-01-18T22:08:26.624+0300] {standard_task_runner.py:55} INFO - Started process 5975 to run task
[2024-01-18T22:08:26.654+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'test_model_exp', 'scheduled__2024-01-01T03:30:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmpsh0_w3sw']
[2024-01-18T22:08:26.658+0300] {standard_task_runner.py:83} INFO - Job 42: Subtask test_model_exp
[2024-01-18T22:08:26.817+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.test_model_exp scheduled__2024-01-01T03:30:00+00:00 [running]> on host xflow
[2024-01-18T22:08:26.956+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=test_model_exp
AIRFLOW_CTX_EXECUTION_DATE=2024-01-01T03:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-01T03:30:00+00:00
[2024-01-18T22:08:26.961+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-18T22:08:26.963+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/angry/mlops_4/scripts/test_model.py exp']
[2024-01-18T22:08:26.991+0300] {subprocess.py:86} INFO - Output:
[2024-01-18T22:08:35.779+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-18T22:08:35.867+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=test_model_exp, execution_date=20240101T033000, start_date=20240118T190826, end_date=20240118T190835
[2024-01-18T22:08:35.917+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-18T22:08:35.943+0300] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
